libres/lib/CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ pybind11_add_module(
   # IES_ENKF
   analysis/ies/ies_enkf.cpp
   analysis/ies/ies_enkf_config.cpp
-  analysis/ies/ies_enkf_data.cpp
+  analysis/ies/ies_enkf_state.cpp
```

Comment:
Is this a "iterative ensemble smoother ensemble Kalman filter state" module?

Reply:
What do you want changed?

The point is that the struct `ies_enkf_data` contains state about the update which has (close to) application lifetime; this long lasting state will create complexity and to highlight that is indeed state I have suggested a rename from `data` to `state`.

Reply:
If it's not an iterative ensemble smoother ensemble Kalman filter state I would like it renamed to something more sensible when we're renaming it anyway.

Reply:
Well - the whole subsystem/module is already prefixed with `ies_enkf` - does not make sense to only change one of the parts to something else. Could of course rename everything - then I would suggesting removing the `enkf` part.

Reply:
Right. It's just that the "data" to "state" change does not improve anything in my feeble mind. But I know that a diff has a cost, so does a PR. So weighing cost vs benefit here makes it a bit unclear whether or not we need this. 

Anyway, I'll dismiss this review shortly and let you decide.

Reply:
> It's just that the "data" to "state" change does not improve anything in my feeble mind

Well - there we disagree. `data` has a connotation in enkf context, in this particular example the word `data` is *not* used according to that connotation.

Reply:
Any documentation for this so that I can read up?

Reply:
Well not really - e.g. Geirs book; but thinking of it also `state` has connotations in that literature. I'll just close this.